bot/exts/fun/quack.py

```diff
@@ -0,0 +1,62 @@
+from typing import Optional
+
+import discord
+from discord.ext import commands
+
+from bot.bot import Bot
+from bot.constants import Colours
+
+API_URL = 'https://quackstack.pythondiscord.com'
+
+
+class Quack(commands.Cog):
+    """Cog for the .quack command."""
+
+    def __init__(self, bot: Bot) -> None:
+        self.bot = bot
+
+    @commands.command()
+    async def quack(self, ctx: commands.Context, ducktype: Optional[str] = "duck", seed: Optional[str] = None) -> None:
+        """
+        Uses the Quackstack API to generate a random duck.
+
+        If a seed is provided, a duck is generated based on the given seed.
+        If a duck type is provided, either a duck or manduck will be generated.
+        """
+        ducktype = ducktype.lower()
```

Comment:
Even though this doesn't adversely affect the function in this case, this is a case of mutating default arguments (I think?), check out `!mutable-default-args` in PyDis

Reply:
No, strings are immutable. Besides, even if they were this would not hurt anything.